### PR TITLE
Image promotion for cluster-api-azure v1.11.5

### PR DESCRIPTION
**Pull request by nojnhuh**
Image promotion for cluster-api-azure v1.11.5
This is an automated PR generated from `kpromo`
```
kpromo pr --fork nojnhuh --project cluster-api-azure --reviewers "@CecileRobertMichon @jackfrancis @mboersma @nojnhuh @sonasingh46" --tag v1.11.5
```

/hold
cc: @CecileRobertMichon @jackfrancis @mboersma @nojnhuh @sonasingh46

### CODE REVIEW

**Review by CecileRobertMichon — COMMENTED** [review]
/lgtm

**Comment by jackfrancis** [comment]
@nojnhuh can you link to the build job that created this image so I can sanity check the hash reference?

File: registry.k8s.io/images/k8s-staging-cluster-api-azure/images.yaml [hunk 1]
```diff
@@ -43,6 +43,7 @@
     "sha256:e4fdc39a09eb48de3f5e29bc61ab763b539a1f394bcfa538dcffb8dbe1335fbc": ["v1.11.2"]
     "sha256:fbc09e642af090af9b2ed240a250eddef354f5f2a8781d58ecf3668c2e5f38e2": ["v1.11.3"]
     "sha256:27d3742505c1c7f23468c7d08a47f29e184e27d13b535d12920e587f578e8e54": ["v1.11.4"]
+    "sha256:af8b5daff0fc182cd9d51920bb0a9ae18c0ab219d894ddcda68f61762a285d6c": ["v1.11.5"]
```
Comment (nojnhuh) [inline registry.k8s.io/images/k8s-staging-cluster-api-azure/images.yaml:46]: Looks good :+1:
https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api-azure/global/cluster-api-azure-controller@sha256:af8b5daff0fc182cd9d51920bb0a9ae18c0ab219d894ddcda68f61762a285d6c/details

/hold cancel
/assign @CecileRobertMichon

**Pull request merged**